project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
-addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
+addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.6")
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.10")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2-1")
\ No newline at end of file
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0")
```

Comment:
Does it still work with the major version bump?

Reply:
There's no way to know before an actual release, but that works with other projects we've upgraded :)